potku.py

```diff
@@ -1469,22 +1471,37 @@ def __tab_exists(self, tab_id):
     def __open_manual(self):
         """Open user manual.
         """
+        manual_filename = gf.get_root_dir() / "documentation" / "Potku-User-Manual.pdf"
+        self.__open_file(manual_filename)        
+                
+                
+    def __open_data_help(self):
```

Comment:
```suggestion
        self.__open_file(manual_filename)        

    def __open_data_help(self):
```
Code style fix:
- PEP 8: E303 too many blank lines (2)